modules/serverless-kafka-broker-configmap.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * /serverless/admin_guide/serverless-kafka-admin.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-kafka-broker-configmap_{context}"]
+= Configuring Kafka broker settings
+
+You can configure the number of topic partitions, replication factor, and bootstrap servers for a Kafka broker, by creating a config map and referencing this config map in the Kafka `Broker` object.
+
+:FeatureName: Kafka broker
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+
+* You have cluster or dedicated administrator permissions on {product-title}.
+* The {ServerlessOperatorName}, Knative Eventing, and the `KnativeKafka` custom resource (CR) are installed on your {product-title} cluster.
+* You have created a project or have access to a project with the appropriate roles and permissions to create applications and other workloads in {product-title}.
+* You have installed the OpenShift (`oc`) CLI.
+
+.Procedure
+
+. Modify the `kafka-broker-config` config map, or create your own config map that contains the following configuration:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <config_map_name> <1>
+  namespace: <namespace> <2>
+data:
+  default.topic.partitions: <integer> <3>
+  default.topic.replication.factor: <integer> <4>
+  bootstrap.servers: <list_of_servers> <5>
+----
+<1> The config map name.
+<2> The namespace where the config map exists.
+<3> The number of topic partitions for the Kafka broker. This controls how quickly events can be sent to the broker. A higher number of partitions requires greater compute resources.
+<4> The replication factor of topic messages. This prevents against data loss. A higher replication factor requires greater compute resources and more storage.
+<5> A comma separated list of bootstrap servers. This can be inside or outside of the {product-title} cluster, and is a list of Kafka clusters that the broker receives events from and sends events to.
++
+[IMPORTANT]
+====
+The `default.topic.replication.factor` value must be less than or equal to the number of Kafka broker instances in your cluster. For example, if you only have one Kafka broker, the `default.topic.replication.factor` value should not be more than 1.
+====
++
+.Example Kafka broker config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kafka-broker-config
+  namespace: knative-eventing
+data:
+  default.topic.partitions: "10"
+  default.topic.replication.factor: "3"
+  bootstrap.servers: "my-cluster-kafka-bootstrap.kafka:9092"
+----
+
+. Apply the config map:
++
+[source,yaml]
+----
+$ oc apply -f <config_map_filename>
+----
+
+. Specify the config map for the Kafka `Broker` object:
++
+.Example Broker object
+[source,yaml]
+----
+apiVersion: eventing.knative.dev/v1
+kind: Broker
+metadata:
+  name: <broker_name> <1>
+  namespace: <namespace> <2>
+  eventing.knative.dev/broker.class: Kafka <3>
```

Comment:
```suggestion
  annotations:
    eventing.knative.dev/broker.class: Kafka <3>
```

This should be an annotation, same as here: https://knative.dev/docs/eventing/configuration/broker-configuration/#channel-implementation-options